RedHatStyleGuide/en-US/Release_Notes.ent

```diff
@@ -1,5 +1,5 @@
 <!ENTITY PRODUCT "Red Hat Style Guide">
 <!ENTITY BOOKID "Writing_Style_Guide">
-<!ENTITY YEAR "2021">
+<!ENTITY YEAR "2022">
 <!ENTITY HOLDER "Red Hat, Inc">
-<!-- Change entity year to 2021 -->
\ No newline at end of file
+<!-- Change entity year to 2022 -->
```

Comment:
```suggestion
```
You don't need to comment what you changed. It shows up in the diff. Not an issue in this file but in other files it can also get messy and make the actual content hard to read.

Reply:
This comment was already there. Now removing it.